sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -8269,10 +8280,14 @@ public void testQueryWithSelectProjectAndIdentityProjectDoesNotRename() throws E
     cannotVectorize();
     requireMergeBuffers(3);
     testQuery(
-        PLANNER_CONFIG_NO_HLL.withOverrides(ImmutableMap.of(
-            PlannerConfig.CTX_KEY_USE_GROUPING_SET_FOR_EXACT_DISTINCT,
-            "true"
-        )),
+        PLANNER_CONFIG_NO_HLL.withOverrides(
+            new QueryContext(
+                ImmutableMap.of(
+                    PlannerConfig.CTX_KEY_USE_GROUPING_SET_FOR_EXACT_DISTINCT,
+                    "true"
+                )
+            )
```

Comment:
Nit: Maybe add a private util method for this? This code is present in 4 places.

Reply:
I'm not sure if that is worth. I think we rather need to refactor and split `CalciteQueryTest` since this class is too big now.

Reply:
+1 on splitting up the `CalciteQueryTest`